src/wallet.js

```diff
@@ -67,24 +67,28 @@ const wallet = {
    *
    * @param {string} words Words (separated by space) to generate the HD Wallet seed
    *
-   * @return {Object} {'valid': boolean, 'message': string}
+   * @return {Object} {'valid': boolean, 'message': string, 'words': cleanWords}
```

Comment:
`words` is type `string`, right?

Also, it might be more useful to just return an array, no?

Reply:
Yeah, I improved this comment. Thanks